### ISSUE TITLE
cannot use r.db (type *"gorm.io/gorm".DB) as type *"github.com/jinzhu/gorm".DB

### ISSUE THREAD

**Issue by mattn** [tusharsoni/copper#1]
I'm trying to build tutorial app using copper but following error occured.
```go
func (r *PostsRepo) GetPostByID(ctx context.Context, id string) (*Post, error) {
	db := csql.GetConn(ctx, r.db)
	fmt.Println(db)
	return nil, nil
}
```

```
.\main.go:30:27: cannot use r.db (type *"gorm.io/gorm".DB) as type *"github.com/jinzhu/gorm".DB in argument to csql.GetConn
```

**Comment by tusharsoni** [comment]
Hi @mattn, can you make sure that you are using `github.com/jinzhu/gorm` in your import statements and try again?

I believe the gorm repository has moved to use `gorm.io/gorm` but Copper has not been updated yet to use that yet.

**Comment by mattn** [comment]
You use github.com/jinzhu/gorm here.

https://github.com/tusharsoni/copper/blob/7d4e7be0368e2c3e575b6dc89ff33f3296ac6a43/csql/sql.go#L9

**Comment by mattn** [comment]
Also here

https://github.com/tusharsoni/copper/blob/7d4e7be0368e2c3e575b6dc89ff33f3296ac6a43/csql/conn.go#L6

**Comment by mattn** [comment]
Ah, sorry. I did not read your mention.

> I believe the gorm repository has moved to gorm.io/gorm but Copper has not been updated to use that yet.

Yes, I'm waiting the fix. Thank you. :)

**Comment by tusharsoni** [comment]
This should now be fixed in ffe3eeb10fa01b35cd165a607921173467b2b2a9!

**Comment by mattn** [comment]
Thank you!
